package.json

```diff
@@ -33,23 +34,9 @@
         "@types/react-dom": "^18.0.0",
         "@types/react-router-dom": "^5.1.6",
         "@types/react-virtualized": "^9.21.21",
-        "@typescript-eslint/eslint-plugin": "^5.8.0",
-        "@typescript-eslint/parser": "^5.8.0",
         "babel-polyfill": "^6.26.0",
         "dotenv-webpack": "^8.0.1",
-        "eslint": "^7.18.0",
-        "eslint-config-airbnb-base": "^14.2.1",
-        "eslint-config-binary": "^1.0.2",
-        "eslint-config-prettier": "^7.2.0",
-        "eslint-formatter-pretty": "^4.0.0",
-        "eslint-import-resolver-typescript": "^3.5.5",
-        "eslint-import-resolver-webpack": "^0.13.0",
-        "eslint-plugin-import": "^2.23.4",
-        "eslint-plugin-jest-dom": "^3.8.1",
-        "eslint-plugin-prettier": "^3.3.1",
-        "eslint-plugin-react": "^7.22.0",
-        "eslint-plugin-react-hooks": "^4.2.0",
-        "eslint-plugin-testing-library": "^4.2.0",
+        "eslint-plugin-localize-usage": "^1.0.6",
```

Comment:
Had to install `eslint-plugin-localize-usage` because `@deriv/eslint-config-deriv` was dependent on it:
![Screenshot 2023-07-12 at 3 11 12 PM](https://github.com/binary-com/deriv-app/assets/103016120/2206295e-53b4-46d5-9695-55f1fde6f5db)